Improve diff selection box

cintek opened the pull request:
- prevent changing width (Solves #218)
- work better on small screens
- remove error messages about failures regarding temporary documents when uploading a new document
- add tracking ID so it's easier for users to distinguish between the documents in the box

JanHoefelmeyer reviewed (changes requested):
Everything works as advertised, but the default now requires scrolling to select temporary documents for comparisons. 

Also the tooltip for the comparison button for temporay documents is broken declaring undefined as document, but this is consistent with main and not introduced with this PR

cintek commented:
> Everything works as advertised, but the default now requires scrolling to select temporary documents for comparisons. Maybe consider putting the + sign to the left, even if it is less consistent?

I moved the icons to the left. Maybe we should also do this in other areas like Search because there we also have this kind of problem when the table isn't fitting into the screen.

> Also the tooltip for the comparison button for temporay documents is broken declaring undefined as document, but this is consistent with main and not introduced with this PR

I fixed it anyway and made the tooltips of both icons consistent (before, both tried to show something different).

JanHoefelmeyer reviewed (approved):
LGTM & works
